plugins/main/bridge/bridge.go

```diff
@@ -58,12 +58,14 @@ type NetConf struct {
 	PromiscMode  bool   `json:"promiscMode"`
 	Vlan         int    `json:"vlan"`
 	MacSpoofChk  bool   `json:"macspoofchk,omitempty"`
+	OmitDad      bool   `json:"omitdad,omitempty"`
```

Comment:
One thing I am noticing is that accept_dad is an integer that has 0,1,2 values. I am not sure how that escaped me. I will need to resolve that